src/main/scala/io/iohk/ethereum/domain/SignedTransaction.scala

```diff
@@ -94,6 +96,25 @@ object SignedTransaction {
       sender <- SignedTransaction.getSender(tx, signature, recoveredPointSign)
     } yield SignedTransaction(tx, signature, sender)
   }
+
+  def sign(tx: Transaction, keyPair: AsymmetricCipherKeyPair): SignedTransaction = {
+    val bytes = crypto.kec256(
```

Comment:
I'm wondering, should we obtain the bytes to sign as in `getSender` (https://github.com/input-output-hk/etc-client/blob/86c57a8ed9a702b67a18cbbc9b5f01a3121c2c3b/src/main/scala/io/iohk/ethereum/domain/SignedTransaction.scala#L45)? At least EthereumJ seems to do so in that way, or maybe I'm missing something